test/compflags/bradc/printstuff/version.goodstart

```diff
@@ -1 +1 @@
- version 2.1.0
\ No newline at end of file
+ version 2.2.0
```

Comment:
I think you need to remove the trailing line break here to get the tests to pass

Reply:
Yes, that is correct and I could only get this working in vi...    

```
[Test Summary - 240626.174407]
[Summary: #Successes = 10 | #Failures = 0 | #Futures = 0 | #Warnings = 0 ]
[Summary: #Passing Suppressions = 0 | #Passing Futures = 0 ]
[END]
```